server/src/api/requests.ts

```diff
@@ -88,6 +72,26 @@ export function nestedRequest(request: KnexRequest, isAdmin: boolean) {
     };
 }
 
+export function redactedItem(item, isAdmin: boolean, qtyInStock: ItemQtyAvailable, qtyAvailableForApproval: ItemQtyAvailable, qtyUnreserved: ItemQtyAvailable) {
+    return {
```

Comment:
is this something you prefer over redacting given keys if non-admin? i would expect default bhvr of adding fields somewhere in the model to be piped to clientside, but this is another place you have to update

Reply:
Yeah, barring adding a proper permissions library, this is the easiest solution